src/main/java/bookstore/model/Book.java

```diff
@@ -23,14 +23,14 @@ public class Book {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @NonNull
+    @NotNull
     private String title;
-    @NonNull
+    @NotNull
     private String author;
-    @NonNull
+    @NotNull
     @Column(unique = true)
     private String isbn;
-    @NonNull
+    @NotNull
```

Comment:
Don’t use validators with Entity. 
For Entity you should use `@Column(nullable = false)`